### PR TITLE
Fix players lighting redstone ore inside claims.

### PR DESCRIPTION
**Pull request by KJNine**
Redstone Ore lights from being interacted with/walked on by players, causing any observers watching it to trigger.
This PR makes it require build trust to light.

### CODE REVIEW

File: src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java [hunk 1]
```diff
@@ -1791,7 +1792,7 @@ else if (clickedBlock != null && instance.config_claims_preventTheft && clickedB
             }
         }
 
-        //apply rule for note blocks and repeaters and daylight sensors //RoboMWM: Include flower pots
+        //apply rule for note blocks and repeaters and daylight sensors //RoboMWM: Include flower pots //PR: Include redstone ore
```
Comment (Jikoo) [inline src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java]: The comment here identifying a behavior change from Robo is from before TechFortress was the official maintainer of GP iirc - would be better to rephrase the comment than keep adding junk.

**Review by Jikoo — COMMENTED** [review]
See the pressure plate section in the [FAQ](https://github.com/TechFortress/GriefPrevention/issues/752). Monsters and dropped dropped items also activate redstone ore. This is an incomplete solution that will likely result in even more confusion when users falsely assume that they're safe.

File: src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java (hunk 1 above)
Comment (KJNine) [reply src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java:1795]: Makes sense, however similar to pressure plates, it still makes sense to protect it from players. For example. a redstone ore that is in the wall can only be activated by players, and could be used as a trigger for redstone doors, etc.

**Comment by KJNine** [comment]
Makes sense, however similar to pressure plates, it still makes sense to protect it from players. For example. a redstone ore that is in the wall can only be activated by players, and could be used as a trigger for redstone doors, etc.

**Comment by RoboMWM** [comment]
I'm also against this - current behavior is consistent with other stuff that gets activated (aside from wooden buttons perhaps - protections for that should/will be dropped from that since you can use an arrow to trigger those). Also, makes no sense for build trust, would be at best for accesstrust.

**Comment by KJNine** [comment]
I agree it should be accesstrust, I only used build trust to keep consistent with other redstone types. I also mainly based this off the protection of wooden buttons and plates, since they have similar functionality to redstone ore so redstone ore should have similar protections, but if wooden buttons are already planned to lose protection then it would make sense to not protect redstone ore too.

**Comment by RoboMWM** [comment]
Don't think any type of pressure plate is protected.

**Comment by Jikoo** [comment]
No pressure plates are protected. Wooden buttons are only protected due to an oversight (by me, I think - maybe the 1.16 update instead) during conversion to tags.

**Pull request closed without merging**